Automattic/sensei
# Add a "more" link to the course modules column

Part of #4529

### Changes proposed in this Pull Request

* Change how many course modules column links are displayed.
* Add a filter to modify the number of displayed links.
* Add a "more" link, that shows the rest.

### Testing instructions

* Make a course with 4 or more modules.
* Go to the course index screen in the admin
* Make sure there is a "+x more" link in the modules column for that course.
* Make sure that when you click the link, the other module links are displayed.
* Make a course with 3 or less modules.
* The link should not be displayed for that course.

### New hooks

* `sensei_module_course_column_max_links_count` - The number of modules to be displayed in the course modules column.

### Screenshot / Video

<img width="891" alt="Markup 2022-01-21 at 22 31 46" src="https://user-images.githubusercontent.com/1612178/150596230-bddbd67c-4cf0-42d3-aae6-14ca6fd35783.png">
